src/fonduer/supervision/labeler.py

```diff
@@ -356,30 +361,38 @@ def _after_apply(
 
     def get_gold_labels(
         self, cand_lists: List[List[Candidate]], annotator: Optional[str] = None
-    ) -> List[csr_matrix]:
-        """Load sparse matrix of GoldLabels for each candidate_class.
+    ) -> List[np.ndarray]:
+        """Load dense matrix of GoldLabels for each candidate_class.
 
         :param cand_lists: The candidates to get gold labels for.
         :type cand_lists: List of list of candidates.
         :param annotator: A specific annotator key to get labels for. Default
             None.
         :type annotator: str
-        :return: A list of MxN sparse matrix where M are the candidates and N is the
+        :return: A list of MxN dense matrix where M are the candidates and N is the
             annotators. If annotator is provided, return a list of Mx1 matrix.
-        :rtype: list[csr_matrix]
+        :rtype: list[np.ndarray]
         """
-        return get_sparse_matrix(self.session, GoldLabelKey, cand_lists, key=annotator)
+        return [
+            m.toarray() - 1  # convert to {-1, 0, ..., k-1}
+            for m in get_sparse_matrix(
+                self.session, GoldLabelKey, cand_lists, key=annotator
+            )
+        ]
 
```

Comment:
Can we make a wrapper function in utils for this?

Reply:
Implemented at 7fc08d985a65e08ff117d4bc66d372fbd30546e9.